### PR TITLE
[move core modules] simplify TransactionFees genesis setup

### PR DESCRIPTION
**Pull request by sblackshear**
- `Preburn` resource can only be packed during DD account creation
- Kill procedure that allows the holder of the `BurnCapability` to create a `Preburn` resource
- Move all of the logic for setting up the fees account into one place
- Killed `Empty` module, which was unused

## Motivation

(Write your motivation for proposed changes here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/website, and link to your PR here.)

### CODE REVIEW

**Comment by sblackshear** [comment]
> deleted language/move-lang/functional-tests/tests/libra/multiple_preburners.move 

I got rid of this because it's all-but-impossible to test without support for creating accounts with a specific role in the e2e test. We can bring it back once we fix that and/or if we have some dummy DD accounts in genesis (which we might want anyway).

File: language/stdlib/modules/Libra.move [hunk 1]
```diff
@@ -327,15 +317,23 @@ module Libra {
...
-    public fun publish_preburn_to_account<CoinType>(creator: &signer, account: &signer) {
-        Association::assert_account_is_blessed(creator);
-        assert_is_coin<CoinType>();
-        let preburn = Preburn<CoinType> { requests: Vector::empty() };
-        publish_preburn<CoinType>(account, preburn)
+    public fun publish_preburn_to_account<CoinType>(
+        creator: &signer, account: &signer
+    ) acquires CurrencyInfo {
+        Transaction::assert(!is_synthetic_currency<CoinType>(), 202);
+        move_to(account, create_preburn<CoinType>(creator))
```
Comment (tzakian) [inline language/stdlib/modules/Libra.move:336]: Took me a bit to get this...but it's amazing!

File: language/stdlib/modules/TransactionFee.move [hunk 2]
```diff
@@ -30,40 +31,51 @@ module TransactionFee {
...
     /// Called in genesis. Sets up the needed resources to collect
-    /// transaction fees by the `0xB1E55ED` account.
-    public fun initialize(blessed_account: &signer, fee_account: &signer) {
-        Transaction::assert(Signer::address_of(blessed_account) == 0xB1E55ED, 0);
+    /// transaction fees from the `0xFEE` account with the `0xB1E55ED` account.
+    public fun initialize(association: &signer, fee_account: &signer) {
+        Association::assert_is_root(association);
+        Transaction::assert(Signer::address_of(fee_account) == 0xFEE, 0);
+
+        let dummy_auth_key_prefix = x"00000000000000000000000000000000";
```
Comment (tzakian) [inline language/stdlib/modules/TransactionFee.move:39]: Nit: maybe pass the `dummy_auth_key_prefix` in? That way there's one less "magical" constant.

**Pull request closed without merging**